Style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  background-image: url(Header-pic.png);
+  background-repeat: repeat;
+}
+
+header {
+  background-color: aliceblue;
+  height: 40px;
+  display: flex;
+}
+
+.navbar {
+  font-size: large;
+}
+
+nav a {
+  margin: 10px;
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: larger;
+}
+
+.union {
+  width: 40px;
+  height: 40px;
+}
+
+li {
+  display: inline;
+  margin-left: 10px;
+}
+
+.title {
+  padding-top: 100px;
+  padding-left: 40px;
+  font-size: large;
+  text-align: center;
+}
+
+.title .name {
+  color: rgb(23, 11, 69);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+}
+
+.job {
+  padding: 10px;
+  color: rgb(27, 168, 19);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+}
+
+.content {
+  display: flex;
+  padding: 40px;
+  font-size: 20px;
```

Comment:
- [ ] Please as suggested by the previous reviewer and as [required](https://github.com/microverseinc/curriculum-html-css/blob/096130a7c3e71e15c3c97ddc2ed0ec6467621a8c/portfolio/1_setup.md?plain=1#L58) for this project, kindly ensure that the headline section is laid out using flexbox.